spec/models/info_request_spec.rb

```diff
@@ -1510,6 +1485,127 @@
 
   end
 
+  describe 'keeping track of the last public response date' do
+
+    let(:old_date) { Time.now - 21.days }
+    let(:recent_date) { Time.now - 2.days }
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.